From 3ed893c1137d843b75501ff22c8547848bcfca12 Mon Sep 17 00:00:00 2001
From: Joseph Trebbien <jtrebbien@gmail.com>
Date: Wed, 20 Jun 2012 13:44:00 -0400
Subject: [PATCH] Run libtoolize before aclocal in autogen.sh

---
 autogen.sh |    8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/autogen.sh b/autogen.sh
index 2f5077a..f7d5b6a 100755
--- a/autogen.sh
+++ b/autogen.sh
@@ -195,14 +195,14 @@ $znew
 
 fi
 
-echo "Running aclocal $ACLOCAL_FLAGS"
-aclocal $ACLOCAL_FLAGS \
-    || error_and_exit "aclocal failed"
-
 echo "Running libtoolize"
 libtoolize --copy --force \
     || error_and_exit "libtoolize failed"
 
+echo "Running aclocal $ACLOCAL_FLAGS"
+aclocal $ACLOCAL_FLAGS \
+    || error_and_exit "aclocal failed"
+
 # optional feature: autoheader
 (autoheader --version) < /dev/null > /dev/null 2>&1
 if [ $? -eq 0 ]; then
-- 
1.7.10.2
